### ISSUE TITLE
[PROJECT] Improve translated content authorship

### ISSUE THREAD

**Issue by Rumyra** [mdn/mdn#428]
From @queengooborg

TLDR, the scope of this content is to **format > cleanup > update** our current localized docs.  This should greatly improve the translated documentation and pave a way forward for future localization.

The plan is the following:

- Fix all formatting, linting, etc. that can be fixed automatically:
  - Format all applicable content using Prettier (which should also improve our ability to synchronize with upstream)
  - Synchronize Markdownlint rules with upstream and fix linting errors that come from such rules
  - Introduce additional Markdownlint rules as needed for specific locales
  - Fix the flaws that can automatically be fixed (links with redirects, for example)
- Check for untranslated and partially translated content, and take care of them
  - For fully untranslated content, and for content that’s less than ~10%(?) translated…
    - Delete the files if we can, so that we can translate them at a later date
  - For content that’s partially translated…
    - Re-synchronize and translate them
- Finally, go through the remaining files that don’t have a source commit set
  - Check to see what needs to be done to synchronize them, and do so
  - Run through files in ranking of (English) popularity

**Comment by SphinxKnight** [comment]
@mdn/localization-team-leads @mdn/yari-content-es @mdn/yari-content-fr @mdn/yari-content-ja @mdn/yari-content-ko @mdn/yari-content-pt-br @mdn/yari-content-ru :)

**Comment by hochan222** [comment]
> Note: some localization teams have recommended keeping the documents and filing an issue instead, to keep redirects and to keep contributor motivation

Where should I create the folder name and folder location? The current plan is to move it to `files/ko/orphaned` or create a new folder `files/ko/fully_untranslated` tell me your opinions.

**Comment by queengooborg** [comment]
> Where should I create the folder name and folder location? The current plan is to move it to `files/ko/orphaned` or create a new folder `files/ko/fully_untranslated`. Are there any expected problems? Tell me your opinions.

Pages that are mostly/fully untranslated shouldn't be moved into another folder. At that point, removal would be much better.  There's no value in retaining untranslated pages in non-English locales (if anything, it actually has negative value).

**Comment by hochan222** [comment]
I plan to remove all pages that are not fully translated. However, there are pages with only partial translations and pages with more than 3 paragraphs translated, so I would like to keep them somewhere for reference for future page translations.

I thought it was a waste to remove pages that were over 50% translated.

**I agree with you, and if so, I'll keep it in my private repo.**

**Comment by queengooborg** [comment]
> However, there are pages with only partial translations and pages with more than 3 paragraphs translated, so I would like to keep them somewhere for reference for future page translations.
> 
> I thought it was a waste to remove pages that were over 50% translated.

For partially translated pages, we'll leave it up to each localization team to decide what to do with each page that is not fully translated.  It's possible that the existing translations are still valuable, or the page has been completely rewritten and needs to be retranslated from scratch -- in either case, you guys will be able to determine which applies much, much better than those that don't speak your language!
